.github/workflows/samples.yml

```diff
@@ -40,10 +40,24 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v4
-      - uses: actions/setup-java@v4
+      # Set up both Java 8 and 21
+      - name: Set up Java 8 and 21
+        uses: actions/setup-java@v4
         with:
-          java-version: 21
+          java-version: |
+            8
+            21
           distribution: corretto
+
+      # Install events module using Java 8
```

Comment:
Have we verified that we need this change? Does CI still pass when we revert this modification?